pyccel/parser/syntactic.py

```diff
@@ -440,25 +441,6 @@ def _visit_NameNode(self, stmt):
             val = strip_ansi_escape.sub('', stmt.value)
             return Symbol(val)
 
-    def _visit_ImportNode(self, stmt):
```

Comment:
can you put back this method and raise an error if we use it instead of deleting it